src/test/data/entity/responseData.ts

```diff
@@ -73,7 +73,7 @@ export const basePayByInstalmentsData = {
 export const basePayBySetDateData = {
   paymentIntention: {
     paymentOption: PaymentOption.BY_SPECIFIED_DATE,
-    paymentDate: '2050-12-31'
+    paymentDate: MomentFactory.currentDate().add(30, 'days')
```

Comment:
Changing this has caused other tests to fail. please update them or revert this

Reply:
changed back to original date